kenji_crosland/index.js

```diff
@@ -0,0 +1,39 @@
+var express = require('express');
+var app = express();
+var fs = require('fs');
+var bodyparser = require('body-parser');
+var jsonParser = bodyparser.json();
+
+app.get('/notes', function(req, res) {
+  fs.readFile(__dirname + '/data/notes.json', function(err, data){
+    var myObj = JSON.parse(data.toString());
+    res.json(myObj);
+  });
+});
+
+app.post('/notes',jsonParser, function(req, res){
+  var postString = JSON.stringify(req.body);
+    fs.readFile(__dirname + '/data/notes.json', function(err, data){
```

Comment:
Why are you doing a readFile in your POST request?  I am a bit confused by this whole bit, actually...and I think you may have misunderstood the assignment (many people did).

The goal is to be able to make multiple POST requests, to (for example) `/notes/molly_note` or `/notes/kenji_note` and have each POST create a file named after the URL that it was POSTed to.  So the above POSTs would become `molly-note.json` and `kenji_note.json`.  The file content is sent in the POST superagent (or curl) command.  Once there are files to grab, a GET request to the corresponding url path should result in the file contents being sent.  So, a GET to `/notes/kenji_note` will result in the contents of `kenji_note.json` being sent as the response.

Make sense?  If not, come see me and we can work through it together.  And if I have missed what you are trying to do with your code, come walk me through it.